testutil/utils.go

```diff
@@ -106,3 +107,25 @@ func JsonGet(j interface{}, components ...string) interface{} {
 	}
 	return j
 }
+
```

Comment:
Note- It will throw a test error if there is an issue with the test result.

Reply:
Since this is a public function,  please add a comment describing the function and its behavior. In particular, document that it is expected to generate a test Error if it times out.